hasher-matcher-actioner/webapp/src/Settings.jsx

```diff
@@ -3,11 +3,189 @@
  */
 
 import React from 'react';
+import Card from 'react-bootstrap/Card';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Tabs from 'react-bootstrap/Tabs';
+import Tab from 'react-bootstrap/Tab';
+import ListGroup from 'react-bootstrap/ListGroup';
 
 export default function Settings() {
   return (
     <>
-      <h1>Settings</h1>
+      <Tabs defaultActiveKey="signals" id="setting-tabs">
+        <Tab eventKey="signals" title="Signals">
+          <SignalSettingsTab />
+        </Tab>
+        <Tab eventKey="pipeline" title="Pipeline">
+          Todo!
+        </Tab>
+      </Tabs>
+    </>
+  );
+}
+
+function SignalSettingsTab() {
+  return (
+    <>
+      <DataSets />
+      <ExternalSignalSources />
+    </>
+  );
+}
+
+function DataSets() {
+  return (
+    <>
+      <h2 className="mt-4">Datasets</h2>
+      <Row className="mt-3">
+        <DataSet />
+        <Col lg={4} sm={6} xs={12} className="mb-4">
+          <Card className="text-center">
+            <Card.Header className="text-white bg-success">
+              <h4 className="mb-0">Verified Cat Imagery</h4>
+            </Card.Header>
+            <Card.Subtitle className="mt-0 mb-0 text-muted">
+              Shared
+            </Card.Subtitle>
+            <Card.Body className="text-left">
+              <Card.Text>
+                <h5>Labels</h5>
+                <ul>
+                  <li>cat</li>
```

Comment:
Using <Badge>'s here instead of ul/li might look nice... https://react-bootstrap.netlify.app/components/badge/#badges